### PR TITLE
Update links in installation guide

### PR DESCRIPTION
**Pull request by gunadhya**
Fixes https://github.com/scrapy/scrapy/issues/4892  

Insured all the other links are working.

### CODE REVIEW

**Review by wRAR — APPROVED** [review]
thanks!

File: docs/intro/install.rst [hunk 1]
```diff
@@ -265,7 +265,7 @@ For details, see `Issue #2473 <https://github.com/scrapy/scrapy/issues/2473>`_.
 .. _cryptography: https://cryptography.io/en/latest/
 .. _pyOpenSSL: https://pypi.org/project/pyOpenSSL/
 .. _setuptools: https://pypi.python.org/pypi/setuptools
-.. _AUR Scrapy package: https://aur.archlinux.org/packages/scrapy/
+.. _ArchlinuxScrapyPackage: https://www.archlinux.org/packages/community-staging/any/scrapy/
```
Comment (elacuesta) [inline docs/intro/install.rst:268]: Changing this name without changing its occurrences in the text looked strange to me, so I checked and it seems like there are in fact no occurrences at all. Should we remove it?

Comment (gunadhya) [reply docs/intro/install.rst:268]: I agree it doesn't appear to be referenced anywhere. I think we can remove it. Should I make the change?

Comment (elacuesta) [reply docs/intro/install.rst:268]: Yes, please do. Thanks.

**Review by elacuesta — APPROVED** [review]

File: docs/intro/install.rst [hunk 2]
```diff
@@ -72,7 +72,7 @@ please refer to their respective installation instructions:
 * `cryptography installation`_
 
 .. _lxml installation: https://lxml.de/installation.html
-.. _cryptography installation: https://cryptography.io/en/latest/installation/
+.. _cryptography installation: https://cryptography.io/en/latest/installation.html
```
Comment (Gallaecio) [inline docs/intro/install.rst:75]: Can we please switch to InterSphinx to handle this?

If we add an entry for `cryptography` to [`intersphinx_mapping`](https://github.com/scrapy/scrapy/blob/master/docs/conf.py#L283) we should be able to link with something like ```:doc:`cryptography:installation` ```.

Comment (gunadhya) [reply docs/intro/install.rst]: Hi @Gallaecio, This is my first time working with intersphinx so please let me know I made a mistake.

File: docs/conf.py [hunk 3]
```diff
@@ -291,6 +291,7 @@
     'tox': ('https://tox.readthedocs.io/en/latest', None),
     'twisted': ('https://twistedmatrix.com/documents/current', None),
     'twistedapi': ('https://twistedmatrix.com/documents/current/api', None),
+    'cryptography' : ('https://cryptography.io/en/latest/', None),
```
Comment (Gallaecio) [inline docs/conf.py:294]: :lipstick: Let’s keep these entries alphabetical :nerd_face:

**Review by Gallaecio — APPROVED** [review]
Looks good to me!

**Pull request merged**